src/main/scripts/releases/3.4/adhoc/createServiceObtainingRelevantApprovals.js

```diff
@@ -0,0 +1,156 @@
+load("../../../utils/addService.js")
+load("../../../utils/program.js")
+var userId = "4228"
+var serviceName = "Developing farm/project/site management plan"
+var sectionName = "Management plan development"
+var label = "Developing Management Plans"
+var newOutputs = [
+    {formName: "Grants and Others Progress Report", sectionName: sectionName},
+    {formName: "Procurement Output Report", sectionName: sectionName}
+];
+// RDP panel procurements are excluded
+var excludedPrograms = ["Priority Places - Procurements", "Priority Species - Procurements - RDP"]
+
```

Comment:
My understanding of this request is that the excluded programs should be the ones that use the NHT Output Report as these services will be used by programs that need to report on these things but don't have the mandatory header section required by the contract (which appears in the NHT Output Report)
You can query for this by checking the 'config.programServiceConfig.serviceFormName' != 'NHT Output Report'

Reply:
Apart from the above, this looks good.